src/sentry/rules/processing/delayed_processing.py

```diff
@@ -47,7 +47,7 @@
 logger = logging.getLogger("sentry.rules.delayed_processing")
 EVENT_LIMIT = 100
 COMPARISON_INTERVALS_VALUES = {k: v[1] for k, v in COMPARISON_INTERVALS.items()}
-CHUNK_BATCH_SIZE = 10000
+CHUNK_BATCH_SIZE = options.get("delayed_processing.batch_size")
```

Comment:
We can't set this as a module variable - it'll never update (and might cause problems on load). We just need to perform the check in function

Reply:
heh, ty, i was just about to ask to check if did that right. thanks!

Reply:
okay, just updated again, anything else i'm missing with the options? (I was looking at the docs here: https://develop.sentry.dev/backend/options/ to add it). ps, thanks for the help!